.github/libs/GitUtils.js

```diff
@@ -7,16 +7,16 @@ const sanitizeStringForJSONParse = require('./sanitizeStringForJSONParse');
  *
  * @param {String} fromRef
  * @param {String} toRef
- * @returns {Promise<Object<{commit: String, subject: String}>>}
+ * @returns {Promise<Array<Object<{commit: String, subject: String, authorName: String}>>>}
  */
-function getMergeLogsAsJSON(fromRef, toRef) {
-    const command = `git log --format='{"commit": "%H", "subject": "%s"},' ${fromRef}...${toRef}`;
```

Comment:
This command was just here for logging purposes, which was annoying because I kept changing it and then being confused why nothing was happening. So instead I log the actual command that we run further down.